Studio.cpp

```diff
@@ -0,0 +1,47 @@
+#include "Studio.h"
+
+Studio::Studio() : numMics(8), numClients (10)
+{
+    hoursPerMix = 4.5f;
+    mixesPerClient = 2;
+}
+
+void Studio::checkEachMic(int numMics_)
+{
+    int i = 1;
+    while(i <= numMics_)
+    {
+        std::cout << "Mic number " << i << ": Signal has been checked" << std::endl;
+        i++;
+    }
```

Comment:
Everywhere that you have `i++`, `i--`, `i -= 1;` or `i += 1;`, you should use `pre-increment/decrement`. 
You are currently doing `post-increment/decrement`.  
you can think of post- and pre- increment like this:
```
int preIncrement(int& i)
{
    i += 1;
    return i; //both the returned value AND i have changed
}
​
int postIncrement(int& i)
{
    int oldVal = i;
    i += 1;
    return oldVal; //i has changed but the value being returned does not reflect this change.
}
```
more information can be found here explaining why `pre-increment/decrement` is preferred over `post-increment/decrement`: https://en.cppreference.com/w/cpp/language/operator_incdec
- [ ] revise